### PR TITLE
Adjust scripts to fit newer OCP versions

### PR DESCRIPTION
**Pull request by kobi86**
Adjust MachineConfig scripts to fit newer OCP versions

### CODE REVIEW

**Comment by kobi86** [comment]
@phoracek @RamLavi

File: mco_ovs_supervisor.yml [hunk 1]
```diff
@@ -10,8 +10,6 @@ spec:
       version: 3.1.0
     systemd:
       units:
-        - enabled: true
```
Comment (phoracek) [inline mco_ovs_supervisor.yml:13]: Awesome! That probably means we can remove the external-ovs labels too.

**Review by phoracek — APPROVED** [review]
Looks good. Waiting for @RamLavi's ack

File: mco_ovs_supervisor.yml (hunk 1 above)
Comment (kobi86) [reply mco_ovs_supervisor.yml:13]: Yep, I will do it in the next PR with the fix of the "setup-ovs-with-labels"

**Review by RamLavi — APPROVED** [review]
/lgtm

**Pull request merged**